arangod/Cluster/DBServerAgencySync.cpp

```diff
@@ -164,6 +164,14 @@ DBServerAgencySyncResult DBServerAgencySync::execute() {
   auto plan = clusterInfo->getPlan();
   auto serverId = arangodb::ServerState::instance()->getId();
 
+  if (plan == nullptr) {
+    // TODO increase log level, except during shutdown?
```

Comment:
I would only increase log level, when the customer can do anything about it. if there is no plan / current, we're just not able to do anything. the situation is that nothing is operating anyhow as there is no agency. so leave the log level at DEBUG.